content/en/docs/contribute/suggesting-improvements.md

```diff
@@ -24,7 +24,7 @@ of the Kubernetes community open a pull request with changes to resolve the issu
 
 If you want to suggest improvements to existing content, or notice an error, then open an issue.
 
-1. Go to the bottom of the page and click the **Create an Issue** button. This redirects you
+1. Click the **Create an Issue** link on the right sidebar. This redirects you
```

Comment:
Hi, @shuuji3 . Thanks for the update.
Would you update the text to, Create an issue?

Reply:
I didn't notice that. I've fixed it!